README.md

```diff
@@ -251,7 +251,7 @@ Templates to help you get started with building a Cosmos SDK blockchain.
 * [REStake](https://restake.app) - Auto-compounder app for Cosmos blockchains using Authz ([source](https://github.com/eco-stake/restake)).
 * [Cosmfaucet](https://github.com/scalalang2/cosmfaucet) - Self-hosted faucet service for Cosmos based blockchain.
 * [Cosmos Notifier](https://cosmos-notifier.decrypto.online) - Governance notification tool for Telegram and Discord ([source](https://github.com/shifty11/cosmos-notifier)).
-* [IBC Anywhere](https://ibc.reece.sh/) - IBC token transfers including multi-chain hops.
+* [Skip:Go](https://go.skip.build) - IBC token transfers, automated swapping and multi-hop routing
```

Comment:
Is it open source?

Reply:
https://github.com/skip-mev/skip-go-app

Reply:
Could it be added in here then, like the others.
Additionally could you fix the lint